app/gateway/changelog.md

```diff
@@ -408,6 +408,68 @@ hang when attempting to expand an API.
   of the `http-log` plugin and the `endpoint` field of the `opentelemetry` plugin,
   do not accept reference values due to incorrect field validation.
 
+## 3.3.1.1
+**Release Date** 2023/10/12
+
+### Breaking Changes
+
+* **Ubuntu 18.04 support removed**: Support for running Kong Gateway on Ubuntu 18.04 ("Bionic") is now deprecated,
```

Comment:
```suggestion
* **Ubuntu 18.04 support deprecated**: Support for running Kong Gateway on Ubuntu 18.04 ("Bionic") is now deprecated,
```

Reply:
Support has been removed, not deprecated. Deprecated would be it's still available but not supported; removed means it doesn't exist.